### ISSUE TITLE
New design might confuse people, questions on verification/endorsement

### ISSUE THREAD

**Issue by jayfk** [djangopackages/djangopackages#503]
@jezdez raised a valid point on Twitter, quoting:

> To be clear, with confusion I mean confusing to users that Django could be endorsing the content of http://djangopackages.org , or that the packages are somehow verified. Same reason djangosnippets doesn’t use color scheme.

**Comment by jayfk** [comment]
To lift some immediate pressure from this, I've added a disclaimer for the time being.

<img width="934" alt="screen shot 2018-03-07 at 09 43 43" src="https://user-images.githubusercontent.com/2930472/37082274-61ea7f64-21ec-11e8-94cd-e7d81bdc6bc9.png">

**Comment by jezdez** [comment]
To clarify a bit (/cc @frankwiles):

When we worked on the new design for the Django site we (DSF board, volunteers and paid contractors) took great pains to create a unique design of the site, funded the design and development of a [styleguide](https://www.djangoproject.com/styleguide/) and generally worked under the assumption that getting the site design right was important for a lasting branding of the Django project.

I'm dismayed to see the design being applied to a related but not identical site without consideration of the effects on the Django site. As such I strongly believe Djangopackages.org should instead have an own design language to clarify its unique selling point and non-existing relationship to the Django project.

In the past we fought hard (years!) to get volunteers to do design work for Django and eventually settled on paying a designer to finish the previously half-finished redesign. This was a non-trivial amount of work, even for the volunteers afterwards implementing the redesign. I personally spent weeks on this and hope you understand that personal perspective on this.

One of the things we decided at the time was to prevent lowering the impact of the design by using it on other site that are related to Django, like for example djangosnippets.org. Even if the snippets site was (and still is) an integral part of the community, applying the same style guide to it would imply an endorsement by the Django project and possibly by the DSF as well and then weakening the ability to recognize the authenticity of the main Django site and documentation. Because the fact was nobody wanted to vouch for the snippets stored there in an official manner.

Another precedence recently was the redesign of the Django admin interface where I also argued to not use the same style guide for it since while it's relatively easy to copy the stylesheet of the site to the admin, it was still not created to cater to the use as a administration backend design. On top of that the implementer of the admin design changes also didn not have permission to reuse the stylesheets in code that is shipped as part of the Django release, effectively relicensing it under the MIT license without a proper disclaimer. I'd like that state the changes in #499 **also don't reference the authors or designers of the original design**.

So to summarize:

Djangosnippets.org should not use parts of the Django site design, since
- (a) it was not created for that purpose
- (b) weakens the design language and branding of the Django site and documentation
- (c) confuses the Django users about the relationship between djangosnippets.org and the Django project and the DSF and
- (d) violates the license agreement of the Django site code.

**Comment by pydanny** [comment]
What about sites like Django weekly (mailing list) or Django Stars (django consultation) that use these colors?

I'm not disagreeing, just wondering where the line starts and ends.

**Comment by jefftriplett** [comment]
Since the Django colors are mentioned on [here](https://www.djangoproject.com/community/logos/) and I don't see where it steers people away from using Django green for their projects. 

While I don't have a strong feeling about this, either way, I do think it's worth clarifying if this is an unwritten rule for Django because historically there's a lot of prior art for projects which do have the Django green in them. However, I do see @jezdez's point as outlined above.

**Comment by jezdez** [comment]
@pydanny I think it's only fair to say that being asked to draw a line, basically making a generically applicable rule, is not the scope of this ticket. The new djangopackages.org design is *very* similar to the djangoproject.com site when viewed side by side, that's my concern here. I haven't reviewed (or have access to) the code of the others sites.

@jefftriplett The [Django logo page](https://www.djangoproject.com/community/logos/) you mention is about the standards we apply when using the official Django logo via the Django Trademark License Agreement, including the colors it has to have.

That the logo is embedded in the Django site with the same color is unrelated to this ticket though and instead about making sure the [styleguide for the Django site](https://www.djangoproject.com/styleguide/) is not used elsewhere. I'm sorry if my wall of text above didn't make that more clear.

**Comment by frankwiles** [comment]
Hi Everyone! 

The DSF Board discussed this at our meeting on Thursday, apologies it's taken me this long to relay the decision. 

After looking things over, while we agree the sites appear similar, we feel they are different enough to avoid confusion.  At least as the design now stands. 

@jayfk we would ask that you: 

- [ ] Properly reference and follow the site's MIT license asap as @jezdez mentioned
- [ ] The current "alert" is fine in terms of language, but we think it distracts from the overall mission of the site.  We'd be fine with moving the alert to say the bottom of the main content area, the footer, making it a light grey text, or otherwise reduce it's visibility a bit.  It should be fairly clear to most that this is the case anyway and while we want to inform we don't really need to shout it! :smile: 
- [ ] Update the FAQ page to reflect that you're the primary maintainer and the DSF funding aspects (we just happened to notice this is way out of date when looking over the site together)

**Comment by jayfk** [comment]
Thanks for the clear instructions @frankwiles, I'm currently deploying the requested changes (commit https://github.com/djangopackages/djangopackages/commit/b7e612a6affad921a23f439cbf69b8e8334f218e).

**New Footer**
<img width="1104" alt="screen shot 2018-03-12 at 09 21 02" src="https://user-images.githubusercontent.com/2930472/37277970-92b83dcc-25e6-11e8-87cb-f2f24dd791ba.png">

-----
> Properly reference and follow the site's MIT license asap as @jezdez mentioned

Can someone help me out with that? Djangopackages is MIT licensed, Djangoproject.com too. What exactly do I need to do here?

**Comment by frankwiles** [comment]
The alert looks great thanks! 

I believe all you need to do is say "Colors and some design elements copied from djangoproject.com with a link to the license"
